be/src/olap/rowset/segment_v2/inverted_index_writer.cpp

```diff
@@ -404,12 +404,15 @@ class InvertedIndexColumnWriterImpl : public InvertedIndexColumnWriter {
                             _parser_type != InvertedIndexParserType::PARSER_NONE) {
                             // in this case stream need to delete after add_document, because the
                             // stream can not reuse for different field
+                            bool own_token_stream = true;
```

Comment:
Only used once

Reply:
yes, to make code more readable.

Reply:
by the way, no need more case, p0 case can cause the memory leak.